docs/reference/data-management.asciidoc

```diff
@@ -22,9 +22,12 @@ It's ok if queries take longer to complete.
 
 To help you manage your data, {es} enables you to:
 
+* Define the retention period of your data via <<index-lifecycle-management, {ilm}>> ({ilm-init}) or via
+<<data-lifecycle-management, {dlm}>> ({dlm-init}), _note {dlm-init} is in_ preview:[]. The retention period is
+the minimum time your data will be stored in {es}. Data older than this period can be deleted by {es}.
```

Comment:
I think it's worth mentioning that everything below this is only relevant to ILM (I was initially confused by the page, and I'm fairly familiar with this).

Reply:
> The docs look great to me (I'll add a couple of comments). But I think we're missing an important one -- `DLM vs ILM`. It seems to me that it could cause a lot of confusion introducing DLM without explaining:
> 
>     * Why did you do this?
> 
>     * What is the difference between DLM and ILM?
> 
>     * How do I know if I should use DLM or ILM?
> 
> 
> And then probably the next batch of questions:
> 
>     * What is happening with ILM (is it going away now)?
> 
>     * Why didn't you just extend ILM with additional functionality?
> 
>     * Will DLM support X in the future (tiers being the big one that comes to mind)?
> 
>     * Just how interoperable are DLM and ILM (the docs touch on this a little bit)?

You are right, this PR doesn't mean to provide a complete DLM documentation. As DLM progresses and matures, there will be a moment that it will be worth to invest in restructuring the documentation to guide users to choose between ILM and DLM. 

For now our goal is to help users that want to experiment with DLM to start with. There is an ongoing effort to create more concise data management documentation so I will add it there too.

@masseyke Are you okay, if we postpone these topics until later?

Reply:
Very good point, I didn't think of that.